addons/jam/CfgMagazineWells.hpp

```diff
@@ -1,71 +1,71 @@
 class CfgMagazineWells {
 
     // Rifle calibre magwells, ordered lexicographically in metric and imperial groups
-    #include "magwells_11x59R.hpp"                  // 11x59mmR Gras | 11mm Vickers
```

Comment:
Why? There was lots of work put into these comments.

Reply:
Indeed, those comments are essential to making sure that things are compatible, not every round is called the same thing everywhere and it's important that people are able to find the correct classes/files to put their mags in.

Reply:
Read the commit title.